helpers/cf_helper.py

```diff
@@ -65,7 +61,7 @@ def stack_was_created_successfully(self, stack_name, attempt=1):
         except Exception as e:
             logging.info('There was a problem checking status of stack: {0}'.format(e))
         logging.info('Stack creation still in progress')
-        time.sleep(20)
+        time.sleep(sleep_time)
```

Comment:
The only reason I added this variable was so that the tests would not need to wait 20 seconds for each attempt. I've added the sleep time to the log as suggested.